services/src/projects/project.rs

```diff
@@ -210,7 +211,7 @@ impl Layer {
     pub fn layer_type(&self) -> LayerType {
         match self.symbology {
             Symbology::Raster(_) => LayerType::Raster,
-            Symbology::Vector(_) => LayerType::Vector,
+            _ => LayerType::Vector,
```

Comment:
I would recommend reverting this change since it is more error-prone to have this as catch all

Reply:
there is no `VectorSymbology` any more because it was inconvient to have `Symbology::Vector(VectorSymbology::Point`. I'd much rather split the `Layer` into a raster and a vector variant like in the frontend. but that's not part of this PR.